docs/_data/sidebar.yml

```diff
@@ -102,18 +102,31 @@
     ja: Apex
   baseUrl: /apex/
   children:
+    -title:
```

Comment:
@sbudhirajadoc I think the lack of a space between the - and title is why the build is failing.  There are a few spots to fix in this file

Reply:
wow. thanks!